### ISSUE TITLE
UX for adding participant from sidebar too direct

### ISSUE THREAD

**Issue by PVince81** [nextcloud/spreed#4484]
## Steps to reproduce
1. Start a conversation with many people
2. On the right sidebar in the participant list, type in some search term in the top field
3. Scroll down then click an entry "by mistake" instead of another one (maybe wanting to click on the three dots for initiating a private chat)

### Expected behaviour
Nothing should happen, I'd consider this a simple search result.
If I want to add a participant, there should be a "+" button behind that I can click.

### Actual behaviour
User or group gets added directly after clicking.
The wrong user got added to the conversation, not sure if they are allowed to see it... need to kick that user out quickly.

## Talk app

**Talk app version:** 11.0 pre-alpha (git)

@nextcloud/designers-talk

**Comment by nickvergessen** [comment]
Yeah a + button where the … button normally is sounds like a good idea.

**Comment by jancborchardt** [comment]
Since for creating conversations we have the "pick people" flow in the modal, would this also make sense here? It allows you to search, pick some people, remove ones you accidentally added, and then confirm. @ma12-co

**Comment by ma12-co** [comment]
> Since for creating conversations we have the "pick people" flow in the modal, would this also make sense here?

I also thought about that @jancborchardt  but i think that the selection bubbles between the two lists would not work.

I suggest we:
- Limit the search to participants in the conversation (do not allow to add new participants via sidebar search). Possibly also display the search bar only if really needed (e.g. if more than 5 participants are overflowing);
- Add an "add participants" button above the search-box that brings up a dialog that allows you to select-deselect participants to add just like in the new group conversation flow;

![IMG_20201102_161240](https://user-images.githubusercontent.com/26852655/97888106-08995d80-1d2b-11eb-884b-464fd2c397ca.jpg)

**Comment by ma12-co** [comment]
Basically what I propose would look like this for small to medium groups

![IMG_20201102_165353](https://user-images.githubusercontent.com/26852655/97889047-2b784180-1d2c-11eb-8a9f-0743b1f1d718.jpg)
